spec/domain/export/tabular/people/person_row_spec.rb

```diff
@@ -25,10 +25,39 @@
       before { person.update(gender: 'w') }
       it { expect(row.fetch(:sww_salutation)).to eq 'Frau' }
     end
-    
+
     context 'with gender m' do
       before { person.update(gender: 'm') }
       it { expect(row.fetch(:sww_salutation)).to eq 'Herr' }
     end
   end
+
+  describe 'roles' do
+    subject(:roles) { row.fetch(:roles) }
+    before { person.roles.first.update(created_at: Time.zone.local(2024, 10, 29, 13, 37)) }
+
+    context 'with no end date' do
```

Comment:
Should there be a context without start date? Or does the start date need to be always present?